templates/al2/template.json

```
@@ -219,6 +229,57 @@
         "WORKING_DIR={{user `working_dir`}}"
       ]
     },
+    {
+      "type": "shell",
+      "script": "./scripts/install-efa.sh",
+      "environment_vars": [
+        "AWS_ACCESS_KEY_ID={{user `aws_access_key_id`}}",
+        "AWS_SECRET_ACCESS_KEY={{user `aws_secret_access_key`}}",
+        "AWS_SESSION_TOKEN={{user `aws_session_token`}}",
+        "BINARY_BUCKET_NAME={{user `binary_bucket_name`}}",
+        "BINARY_BUCKET_REGION={{user `binary_bucket_region`}}",
+        "AWS_REGION={{user `binary_bucket_region`}}"
+      ]
+    },
+    {
+      "type": "shell",
+      "script": "./scripts/install-worker-accelerator-support.sh",
+      "environment_vars": [
+        "ENABLE_ACCELERATOR={{user `enable_accelerator`}}",
+        "ENABLE_EFA={{user `enable_efa`}}"
+      ]
+    },
+    {
+      "type": "shell",
+      "script": "{{template_dir}}/scripts/install-neuron-driver.sh",
+      "environment_vars": [
+        "ENABLE_ACCELERATOR={{user `enable_accelerator`}}"
+      ]
+    },
+    {
+      "type": "shell",
+      "script": "./scripts/install-nvidia-driver-runfile.sh",
```

nit for the filename:
```suggestion
      "script": "./scripts/install-nvidia-driver.sh",
```

the "runfile" aspect is an implementation detail